tools/kdc.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Exit immediately on errors
+set -e
+
+# default values
+ssh_path="${HOME}/.ssh/ccm.pem"
+PRINCIPALS_FILE_NAME=""
+PRINCIPALS_FILE_DIR=""
+
+function usage()
```

Comment:
This duplicates the functionality in `kdc.py`. Could we rather only maintain one of them?